website/src/app/kb/authenticate/oidc/readme.mdx

```diff
@@ -102,13 +98,25 @@ https://your-tenant.okta.com/.well-known/openid-configuration
 
 ### PKCE
 
-If the option is available, be sure to **enable PKCE** for the connector. This
-is a security feature that helps prevent certain types of attacks and is
-recommended for added security.
+The universal OIDC connector **requires** PKCE (Proof Key for Code Exchange) to
+be enabled on the connector. PKCE is a security feature that helps prevent
+certain types of attacks and is recommended for added security.
 
 For more detailed guides specific to each provider, see the
 [Firezone legacy documentation](/docs/authenticate/oidc). Firezone 1.0 uses the
 same OIDC connector under the hood as our legacy version, so the steps should
 similar.
 
+### Provisioning users and groups
+
+Users must be created in Firezone before they can authenticate with the
+universal OIDC connector. You can [create users](/kb/deploy/users) manually in
```

Comment:
This could potentially be confusing to customers that are not on enterprise, because in the portal, you have the option to click on Google, Microsoft, Okta, Jump Cloud, or OIDC even if you're not on the enterprise plan.  All of those options will create a generic OIDC provider behind the scenes but from the user's perspective they may think they aren't using the generic OIDC provider.

Perhaps something like: 
'if you do not have directory sync enabled (enterprise only plans) then you will need to create users before.....'  or something along those lines.

Reply:
Hm, but we have dedicated guides for those connectors. This guide is just for the OIDC connector, i.e. when they click on the `OIDC` connector specifically.

Reply:
Ah, ok, I see.  I was thinking this was on a general 'Authentication' page.  Sorry about that.